identity/src/main/java/com/stripe/android/identity/ui/ConsentScreen.kt

```diff
@@ -228,6 +221,7 @@ private fun SuccessUI(
                 )
             }
 
+            // TODO(ccen-stripe): replace with [Html]
```

Comment:
Note: `AndroidView` is preventing the [performScrollTo](https://developer.android.com/reference/kotlin/androidx/compose/ui/test/package-summary#(androidx.compose.ui.test.SemanticsNodeInteraction).performScrollTo()) from working for unit test, will update the test once we use compose `Text` with [Html](https://github.com/stripe/stripe-android/blob/c16dd94a0ed57ea2dc2dadc6fa8916744fc0b2c9/payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/Html.kt#L62) from `payment-ui-core`